functions/src/wort/core.clj

```diff
@@ -2,6 +2,16 @@
   (:gen-class
    :methods [^:static [handler [String] String]]))
 
+(require '[clojure.string :as str])
+
 (defn -handler
   [s]
   (str "Wort " s " ツ"))
+
+(defn get_words_from_key
+  [payload key]
+  (let [value (filter
+                #(>= (count %) 1) (str/split (get payload key) #"\s+|[.,!?]+" ))]
```

Comment:
This one is actually due to the following situation (see link), cause either the regex
engine in clojure doesn't work the way I think it works, or I'm a bit dense... or both

This snippet will show my confusion

https://repl.it/I4pc/0